File: 백준/Bronze/13416. 주식 투자/주식 투자.js
const input = require('fs').readFileSync(0).toString().trim().split(/\s+/);
let cursor = 0;
const T = Number(input[cursor++]);

for (let t = 0; t < T; t++) {
    const N = Number(input[cursor++]);
    let totalProfit = 0;

    for (let n = 0; n < N; n++) {
        const A = Number(input[cursor++]);
        const B = Number(input[cursor++]);
        const C = Number(input[cursor++]);

        totalProfit += Math.max(A, B, C, 0);
    }
    console.log(totalProfit);
}

File: 백준/Bronze/13416. 주식 투자/주식 투자.py
import sys
input = sys.stdin.readline

T = int(input())

for _ in range(T):
    N = int(input())
    total = 0
    
    for _ in range(N):
        a, b, c = map(int, input().split())
        total += max(a, b, c, 0)
    
    print(total)